packages/core/core/src/AssetGraph.js

```diff
@@ -315,8 +558,13 @@ export default class AssetGraph extends Graph<AssetGraphNode> {
           dependentAssets.push(dependentAsset);
         }
       }
+
+      // TODO ???
+      let assetNode = this.getNode(asset.id) ?? nodeFromAsset(asset);
```

Comment:
Previously, `assetNode.usedSymbols` was being discarded/overwritten - is this the correct solution?